fixes.inc

```diff
@@ -15723,6 +15923,57 @@ native BAD_GetServerVarAsBool(const varname[]) = GetServerVarAsBool;
 	#define GetVehicleSeats__( FIXES_GetVehicleSeats(
 #endif
 
+/**
+ * <remarks>
+ * Checks if component is rear bullbar.
+ * </remarks>
+ */
+
+#if FIX_API
+	stock bool:FIXES_IsComponentRearBullbar(componentid)
+	{
+		if(componentid == 1109 || componentid == 1110)
```

Comment:
Just `return`, no need for `if`.

Reply:
How check then? With ternary operator?

Reply:
No, just return the test.

Reply:
What test? Doesn't get it again.

Reply:
Just

Return

It

...

```c
return (componentid == 1109 || componentid == 1110);
```

Though the whole function isn't needed, so the point is moot.